dingo/gw/result.py

```diff
@@ -506,6 +506,25 @@ def sample_synthetic_phase(
 
         print(f"Done. This took {time.time() - t0:.2f} s.")
 
+    def get_samples_bilby_phase(self):
```

Comment:
I don't see `get_samples_bilby_phase()` getting used anywhere?

Reply:
It is not called internally. It's for the user if they want to convert.